subprojects/plugin-development/build.gradle.kts

```diff
@@ -19,7 +19,6 @@ dependencies {
     implementation(project(":platform-jvm"))
     implementation(project(":reporting"))
     implementation(project(":testing-base"))
-    implementation(project(":testing-jvm"))
```

Comment:
The project dependency in these projects definitely seem to validate this is a step in the right direction.